app/src/main/java/com/example/sharingang/ui/fragments/MessageFragment.kt

```diff
@@ -63,10 +63,10 @@ class MessageFragment : Fragment() {
         setupFields()
         setupSendButton()
         setupUI()
-        messagesLiveData.postValue(listChats)
         lifecycleScope.launch(Dispatchers.Main) {
             setupMessageRefresh()
         }
+        messagesLiveData.postValue(listChats)
```

Comment:
I don't think you need postValue, as this seems to be executed in the main thread, see [https://stackoverflow.com/questions/51299641/difference-of-setvalue-postvalue-in-mutablelivedata](url).
```suggestion
        messagesLiveData.setValue(listChats)
```